pkg/sql/grant_revoke.go

```diff
@@ -58,6 +60,27 @@ func (p *planner) changePrivileges(
 	grantees tree.NameList,
 	changePrivilege func(*sqlbase.PrivilegeDescriptor, string),
 ) (planNode, error) {
+	//check whether grantees exists
+	users, err := GetAllUsers(ctx, p)
+
+	if err != nil {
+		return nil, err
+	}
+	users = append(users, security.RootUser)
```

Comment:
`root` is now part of the users table, you don't need to add it.

Reply:
ok